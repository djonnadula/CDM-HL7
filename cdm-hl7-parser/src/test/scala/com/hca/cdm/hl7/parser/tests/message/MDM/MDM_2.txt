MSH|^~\&||SDCSP|||20160606164544|SEV1254|MDM^T11|EPIC_SDCSP_MDM_122AAS_154123_0_FB74|P|2.3||Epic_CWDV|||||||
EVN|T11|20160606164538||UPDATE_NOTE|SEV1254^CORODBIA^SEVA^M^^^^^AH^^^^^SDCSP
PID||H579625^^^CWTTXIURN^CWTTXIURN|2000524369^^^EPIC^MRN|L151243123^^^SDMC^SDMC~H579625^^^CWTTXIURN^CWTTXIURN|RHONDA^RODMAN^READ||19780808|F|RHONDA^RODMAN^^~RHONDA^RODMAN^R^|W|1 BEND^^DALLAS^TX^78987^USA^P^^TRAVIS|TRAVIS|(999)999-9999^P^H|(999)999-9999^P^W|ENGLISH|D|NON||464-78-9896|||NOT HISPANIC||||||||N
PD1|||CST AUSTIN^^90000019|||||||||||||
PV1||O|CSTA^^^SDCSP^^^^^^^||||WOZR^WOZNIAK^ROBERT^J^^^^^PROVID^^^^PROVID~DVI9907^WOZNIAK^ROBERT^J^^^^^THREEFOUR^^^^THREEFOUR|UNDEFINED^FOUND^PROVIDER^UNKNOWN OR NOT^^^^^PROVID^^^^PROVID~UNK1111^FOUND^PROVIDER^UNKNOWN OR NOT^^^^^THREEFOUR^^^^THREEFOUR|||||||||||32239762||||||||||||||||||||||||||20160606151315|||||||
TXA|1|PROGRESS^Progress Notes|TX|20160606144326|SEV1254^CORODBIA^SEVA^M^^^^^THREEFOUR^^^^THREEFOUR|20160606144326||20160606144326|SEV1254^CORODBIA^SEVA^M^^^^^THREEFOUR^^^^THREEFOUR|||31016394|||||IP||CA|||

