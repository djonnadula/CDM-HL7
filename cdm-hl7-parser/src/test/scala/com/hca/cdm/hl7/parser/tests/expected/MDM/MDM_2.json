{
  "0000.COMN": {
    "sending_facility": "SDCSP",
    "message_control_id": "EPIC_SDCSP_MDM_122AAS_154123_0_FB74",
    "medical_record_num": "2000524369",
    "medical_record_urn": "L151243123",
    "patient_account_num": ""
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_facility": "SDCSP",
    "date_time_of_message": "20160606164544",
    "security": "SEV1254",
    "message_type": {
      "message_code": "MDM",
      "trigger_event": "T11"
    },
    "message_control_id": "EPIC_SDCSP_MDM_122AAS_154123_0_FB74",
    "processing_id": "P",
    "version_id": "2.3",
    "continuation_pointer": "Epic_CWDV"
  },
  "0002.EVN": {
    "event_type_code": "T11",
    "recorded_date_time": "20160606164538",
    "event_reason_code": "UPDATE_NOTE",
    "operator_id": {
      "person_id": "SEV1254",
      "family_name": "CORODBIA",
      "given_name": "SEVA",
      "second_and_further_given_names_or_initials_thereof": "M",
      "assigning_authority": "AH",
      "assigning_facility": "SDCSP"
    }
  },
  "0003.PID": {
    "patient_id": {
      "id_num": "H579625",
      "assigning_authority": "CWTTXIURN",
      "id_type_code": "CWTTXIURN"
    },
    "patient_id_list": {
      "id_num": "2000524369",
      "assigning_authority": "EPIC",
      "id_type_code": "MRN"
    },
    "alternate_patient_id": [
      {
        "id_num": "L151243123",
        "assigning_authority": "SDMC",
        "id_type_code": "SDMC"
      },
      {
        "id_num": "H579625",
        "assigning_authority": "CWTTXIURN",
        "id_type_code": "CWTTXIURN"
      }
    ],
    "patient_name": {
      "family_name": "RHONDA",
      "given_name": "RODMAN",
      "second_and_further_given_names_or_initials_thereof": "READ"
    },
    "date_time_of_birth": "19780808",
    "administrative_sex": "F",
    "patient_alias": [
      {
        "family_name": "RHONDA",
        "given_name": "RODMAN"
      },
      {
        "family_name": "RHONDA",
        "given_name": "RODMAN",
        "second_and_further_given_names_or_initials_thereof": "R"
      }
    ],
    "race": "W",
    "patient_address": {
      "street_address": "1 BEND",
      "city": "DALLAS",
      "state_or_province": "TX",
      "zip_or_postal_code": "78987",
      "country": "USA",
      "address_type": "P",
      "county_parish_code": "TRAVIS"
    },
    "county_code": "TRAVIS",
    "phone_num_home": {
      "telephone_num": "(999)999-9999",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "H"
    },
    "phone_num_business": {
      "telephone_num": "(999)999-9999",
      "telecommunication_use_code": "P",
      "telecommunication_equipment_type": "W"
    },
    "primary_language": "ENGLISH",
    "marital_status": "D",
    "religion": "NON",
    "ssn_num_patient": "464-78-9896",
    "ethnic_group": "NOT HISPANIC",
    "patient_death_indicator": "N"
  },
  "0004.PD1": {
    "patient_primary_facility": {
      "organization_name": "CST AUSTIN",
      "id_num": "90000019"
    }
  },
  "0005.PV1": {
    "patient_class": "O",
    "assigned_patient_location": {
      "point_of_care": "CSTA",
      "facility": "SDCSP"
    },
    "attending_doctor": [
      {
        "person_id": "WOZR",
        "family_name": "WOZNIAK",
        "given_name": "ROBERT",
        "second_and_further_given_names_or_initials_thereof": "J",
        "assigning_authority": "PROVID",
        "id_type_code": "PROVID"
      },
      {
        "person_id": "DVI9907",
        "family_name": "WOZNIAK",
        "given_name": "ROBERT",
        "second_and_further_given_names_or_initials_thereof": "J",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "referring_doctor": [
      {
        "person_id": "UNDEFINED",
        "family_name": "FOUND",
        "given_name": "PROVIDER",
        "second_and_further_given_names_or_initials_thereof": "UNKNOWN OR NOT",
        "assigning_authority": "PROVID",
        "id_type_code": "PROVID"
      },
      {
        "person_id": "UNK1111",
        "family_name": "FOUND",
        "given_name": "PROVIDER",
        "second_and_further_given_names_or_initials_thereof": "UNKNOWN OR NOT",
        "assigning_authority": "THREEFOUR",
        "id_type_code": "THREEFOUR"
      }
    ],
    "visit_num": "32239762",
    "discharge_date_time": "20160606151315"
  },
  "0006.TXA": {
    "set_id": "1",
    "document_type": {
      "id": "PROGRESS",
      "text": "Progress Notes"
    },
    "document_content_presentation": "TX",
    "activity_date_time": "20160606144326",
    "primary_activity_provider_code_name": {
      "person_id": "SEV1254",
      "family_name": "CORODBIA",
      "given_name": "SEVA",
      "second_and_further_given_names_or_initials_thereof": "M",
      "assigning_authority": "THREEFOUR",
      "id_type_code": "THREEFOUR"
    },
    "origination_date_time": "20160606144326",
    "edit_date_time": "20160606144326",
    "originator_code_name": {
      "person_id": "SEV1254",
      "family_name": "CORODBIA",
      "given_name": "SEVA",
      "second_and_further_given_names_or_initials_thereof": "M",
      "assigning_authority": "THREEFOUR",
      "id_type_code": "THREEFOUR"
    },
    "unique_document_num": "31016394",
    "document_completion_status": "IP",
    "document_availability_status": "CA"
  }
}